examples/simple.js

```diff
@@ -11,12 +10,6 @@ var map = new ol.Map({
       source: new ol.source.OSM()
     })
   ],
-  controls: ol.control.defaults({
-    attributionOptions: /** @type {olx.control.AttributionOptions} */ ({
-      collapsible: false
-    })
-  }),
```

Comment:
I always thought that when we are using OSM we must have a non-collapsible attribution, right?